test/controllers/home_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class HomeControllerTest < ActionController::TestCase
+  test "should get home" do
```

Comment:
Awesome, I'll take a look into the Github project as well as the pull requests in Slack.